# DP: Build using the default flags (POWER8) on ppc64el.

--- a/hotspot/make/linux/makefiles/ppc64.make
+++ b/hotspot/make/linux/makefiles/ppc64.make
@@ -47,5 +47,5 @@
   CFLAGS += -DVM_LITTLE_ENDIAN -DABI_ELFv2
 
   # Use Power8, this is the first CPU to support PPC64 LE with ELFv2 ABI.
-  CFLAGS += -mcpu=power7 -mtune=power8 -minsert-sched-nops=regroup_exact -mno-multiple -mno-string
+  CFLAGS += -minsert-sched-nops=regroup_exact -mno-multiple -mno-string
 endif
